six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -253,7 +253,35 @@ class Utilities
     static Vector3 getGroundPlaneNormal(const ComplexData& data);
 
     /*
-     * Parses the XML in 'xmlStream' and converts it into a ComplexData object.
+     * Extracts a Poly1D in the dimention given by index
+     * \param poly The source polynomial
+     * \param index Number specifying dimension. 0-2 for X-Z
+     * \return Poly1D in specified dimension
+     */
+    static Poly1D nPoly(PolyXYZ poly, size_t index);
+
+    /*
+     * Find sign of input, expressed as -1, 0, or 1
+     * \param val A signed number
+     * \return 1 if val is positive, -1 if negative, 0 if 0
+     */
+    template <class T>
```

Comment:
Use `typename` rather than `class`
I think this belongs in the `math` module too
Also, since you know `T` is POD, just pass it in by value